src/posts/2020-10-15-vue-announcer.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Using Vue Announcer'
+description: Silence isn't always golden in web apps
+summary: Vue Announcer provides an easy way to really tell what’s going on in your application to people using screen readers.
+
+categories: [aria, liveregions, screenreader,< spa, html]
```

Comment:
Would you like "liveregions and screenreader" to be a category, or can they be tags?

```yaml
categories: [aria, html, spa]
tags: [liveregions, screenreader]
```

Reply:
Can be tags